addons/test_mail/static/tests/activity_tests.js

```diff
@@ -658,6 +659,50 @@ QUnit.module("test_mail", {}, function () {
         }
     );
 
+
+    QUnit.test("activity view: activity_ids condition not applied as domain search create dialog", async function (assert) {
```

Comment:
can you double check your linter ? 
(you have the eslint rules in `web/tooling/_eslintrc.json`)

![image](https://github.com/odoo/odoo/assets/54316226/1d7c3ded-b752-4da9-897a-8e08165e9daa)